test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -129,13 +129,13 @@ export default class ReleaseSummary extends Component {
                 Object.keys(failedBuildSummary)
                     .sort()
                     .map((buildName) => {
-                        report += failedBuildSummary[buildName];
+                        return (report += failedBuildSummary[buildName]);
                     });
                 report += `${nl} --- ${nl}`;
                 Object.keys(failedTestSummary)
                     .sort()
                     .map((buildName) => {
-                        report += failedTestSummary[buildName];
+                        return (report += failedTestSummary[buildName]);
```

Comment:
We do not need `return` in Line 132 and 138

Reply:
so should we return null??
because if we don't return anything we will get a warning.

Reply:
Personally, I think it is unnecessary to return some value that will not be used in order to avoid the warning message. If you want to fix the warning, I think we should use forEach.